k8s/sonarqube/chart/sonarqube/templates/sonarqube-secret.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{.Release.Name}}-secret
+  labels:
+    app.kubernetes.io/component: {{ .Release.Name }}-secret
+    app.kubernetes.io/name: {{ .Release.Name }}
+type: Opaque
+data:
+  password: "{{.Values.postgresql.db.password}}"
+
```

Comment:
Style: we do not leave empty line before `---`

Reply:
fixed